docs/howtos/customization/setup_export_registry.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_label: "Setting Up An Export Destination Registry"
+sidebar_position: 8
+title: ""
+---
+
+:::note
+
+Do not confuse this topic with external Epinio registries.
+These are where Epinio saves the images of staged applications for its own use.
+See [Setting Up An External Container Registry](setup_external_registry.md) for more information.
+
+Export destination registries on the other hand are where a user saves active applications to,
+for pickup by and use with `helm` and other kubernetes tools.
+
+It does not help that it is perfectly ok to configure the same registry both as external registry
+and as export target.
+
+:::
+
+## How to set up a basic export destination registry
+
+As an operator, replace the bracketed (`<...>`) placeholders in the command below, and then invoke it
+
+```bash
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  annotations:
+    epinio.io/registry-namespace: "<registry-org>"
+  labels:
+    epinio.io/api-export-registry: "true"
+  name: "<destination-name>"
+  namespace: epinio
+stringData:
+  # certs: "<name of cert secret>"
+  .dockerconfigjson: |-
+    {
+      "auths": {
+        "registry.hub.docker.com": {
+          "auth":"PHVzZXI+OjxwYXNzd29yZD4=",
+          "username":"<user>",
+          "password":"<password>"
+        }
+      }
+    }
+EOF
+```
+
+Note that the `auth` element is derived from username and password.
+It has to be replaced as well.
+Its value is the base64 encoding of `<user>:<password>`.
```

Comment:
Must be without a newline character.

Also example would be helpful IMO:
```
echo -n "user:password" | base64 -
```

Reply:
Done, commit 66eba706b6aecb2ed48c75179714241478e78016